phpunit.xml.dist

```diff
@@ -32,7 +32,7 @@ It's auto-generated by sonata-project/dev-kit package.
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
         <env name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml" />
-        <env name="SYMFONY_PHPUNIT_VERSION" value="8.5.0" />
+        <env name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
```

Comment:
We should really change this to avoid being stuck with 8.5.0 while there is a 8.5.8 version

Reply:
Is this configurable via dev-kit? If yes, could you open a PR?

Reply:
Oh yes it is!
https://github.com/sonata-project/dev-kit/blob/master/templates/project/phpunit.xml.dist#L35

Reply:
I created a PR https://github.com/sonata-project/dev-kit/pull/829

Reply:
Please rebase, https://github.com/sonata-project/SonataAdminBundle/pull/6324 is merged

Reply:
Done